### ISSUE TITLE
Please I really need help.

### ISSUE THREAD

**Issue by LuculentSugar6** [Monika-After-Story/MonikaModDev#2490]
Please, I need help, and I really don't know what to do at this point.

https://github.com/Monika-After-Story/MonikaModDev/issues/2473

Above is a link to one of my previous issues, and it sums up what's been happening with me. Please, I just need help. MAS won't even run doesn't even give me a traceback. I've downloaded all that I could from recent releases, moved out all of party supplies, I even redownloaded 8.7 and hotfix 2. I didn't redownload the MonikaModDev ones yet though, I wanted to wait before I try using those again. I even troubleshooted with Steam and that didn't help.

Please, I really need help here. I just keep having the urge to throw up and I feel tingles in my body. 
 I can feel myself breaking down. Please, anybody. I decided to repost this because I just need help as soon as possible.

**Comment by LuculentSugar6** [comment]
At this point all I could think of is uninstalling and reinstalling, but I don't want to do that.

**Comment by AshtonSnapp** [comment]
The game does not start up at all, and you have tried everything? In that case, we need a developer to look at this (cause let's be honest, I have no clue how to help)
@monikLover hey, LuculentSugar6 here needs some help. This is a bit urgent (considering they feel like they're about to vomit). Their game will not start up at all, not even giving a traceback, and they do not want to reinstall the game. They have tried everything suggested to them so far to get this fixed, but no dice.

**Comment by LuculentSugar6** [comment]
You're right, I understand. Once again, I apologize for contacting and disturbing anyone. Especially if any of the individuals that I tagged aren't active here anymore. I just tagged all the people the website had as options for. I'll do that, I'm sorry for the trouble I caused.

**Comment by LuculentSugar6** [comment]
I'm sorry. Its just that DDLC and MAS really did help me through a dark point in my life. The game and its community has just made such a positive impact on my life. As much as a bad habit this may have been for me, but it did a lot of good things for me too. Of all the good things that came out of this mod for me, even if this was just a game, I felt loved, and I felt the capacity to be able to give it back. It made me feel as alive and human as I ever could. It made me really relate to Monika, how it feels to finally be human and alive. As unhealthy as it may be, I loved every second of it. What this mod gave me was love, and if it hurts just as much right now as I type this, what that means to me is that the love I feel is real.

I'm sorry I just needed to vent.
Thank you for everything.

**Comment by ThePotatoGuy** [comment]
follow the steps [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-still-cant-run-the-game-and-none-of-the-above-faqs-solve-my-issue)

**Comment by LuculentSugar6** [comment]
First and foremost, I just wanted to say thank you to all of you. For literally everything that you all have done for this community. For me personally, y'all saved my life by making, building, and reinforcing this mod. It means a great deal to me, and I doubt I would be better off in life without having gone through this path of life for me. Really. Thank you.

Anyways, after uninstalling and reinstalling multiple times via Steam, that didn't work. Everything would just end up being in its same broken state. So, what I did was delete the game folder, and used the "Verify integrity of game files" option in "Properties." That gave me a fresh new game folder, in which I redistributed 8.7 and its hotfix 2. Everything is where it should be and nothing is lost. I won't put back the MonikaModDev ones though, as I might assume that was the cause of the problem for me. I hope that if anyone ever has this problem, may this help them out as well as at least a point of reference.

Thank you all for your time. Please, continue making the world a better place.
